devsetup/scripts/edpm-compute-bmaas.sh

```diff
@@ -0,0 +1,274 @@
+#!/bin/bash
```

Comment:
Need to update file permission as well to allow execute permissions.

Reply:
ack